### ISSUE TITLE
Allow styling text

### ISSUE THREAD

**Issue by timbray** [cheeaun/phanpy#363]
Check out https://void.lgbt/objects/da64778e-033e-43df-9ed8-d91b9ed08ea8

It has bold & italic text, which Phanpy renders nicely. Apparently this is done with a limited subset of HTML tags.  A few Fedi packages, e.g. the Glitch fork and Akkoma, allow authors to style posts (usually with markdown I think).  I think if Phanpy started offering the opportunity to style text, this would really be attractive to a lot of people.

**Comment by paulo-roger** [comment]
Yeah, at least adding the option to set the styling. For instance akkoma has Markdown, Html, BBCode, MFM

**Comment by Fastidious** [comment]
As OP stated, bold and italic are supported. In addition, Phanpy also supports blockquotes, bulleted, and numbered lists. See: https://weaknotes.com/@david/statuses/01J33BWMSHSNAF6J0MXWKHXTG2.

**Comment by Fastidious** [comment]
This is how that status shows in Phanpy:

![image](https://github.com/user-attachments/assets/d03a6696-7465-45a6-a099-bb3a1d2d10f8)

**Comment by Fastidious** [comment]
This is how it renders in Mastodon:

![image](https://github.com/user-attachments/assets/0bd7ea1b-275e-414a-af11-9f6d21dd6441)

**Comment by paulo-roger** [comment]
But in phanpy frontend I have no field to choose if I am posting in plain text or markdown

**Comment by Fastidious** [comment]
@paulo-roger I don't understand. The setting for preferred posting is set at server level. At least, that's what happens on GoToSocial.

**Comment by paulo-roger** [comment]
In akkoma default frontend we can set the global preferred styling, but you can change it in the post field for a particular post if you like. 

In phanpy though none of the methods are presented (unless I am missing where I can set it).

**Comment by paulo-roger** [comment]
![image](https://github.com/user-attachments/assets/2549412a-fac8-4aa4-83a4-202ce40518af)

**Comment by paulo-roger** [comment]
![image](https://github.com/user-attachments/assets/9b62d140-f1a1-494a-b1b4-ab1ae1611621)

**Comment by SadmL** [comment]
> ![image](https://github.com/user-attachments/assets/9b62d140-f1a1-494a-b1b4-ab1ae1611621)


As new Akkoma user, can confirm that Phanpy is not utilizing this setting
